src/main/java/com/paneedah/weaponlib/EntityProjectile.java

```diff
@@ -81,9 +81,9 @@ public EntityProjectile(World world, EntityLivingBase thrower, float velocity, f
     public void setPositionAndDirection(boolean isAim) {
         this.setLocationAndAngles(thrower.posX, thrower.posY + (double) thrower.getEyeHeight(), thrower.posZ, thrower.rotationYaw, thrower.rotationPitch);
         
-        this.posX -= (double) (MathHelper.cos(this.rotationYaw / 180.0F * (float) Math.PI) * 0.16F);
-        this.posY -= 0.10000000149011612D;
-        this.posZ -= (double) (MathHelper.sin(this.rotationYaw / 180.0F * (float) Math.PI) * 0.16F);
+        this.posX -= (double) (MathHelper.cos(this.rotationYaw / 180.0F * (float) Math.PI) * (isAim ? 0F : 0.16F));
+        this.posY -= (isAim ? 0D : 0.10000000149011612D);
+        this.posZ -= (double) (MathHelper.sin(this.rotationYaw / 180.0F * (float) Math.PI) * (isAim ? 0F : 0.16F));
```

Comment:
The fix to this bug should be in a different PR, it is such an important topic I want it to be easily trackable.